### DISCUSSION TITLE
Dockfile manager should detect syntax

### DISCUSSION THREAD

**Discussion by candrews** [renovatebot/renovate#22802] [Suggest an Idea]
### Tell us more.

Renovate should detect `Dockerfile` `syntax` statements which are used to specify the Dockerfile frontend.

See https://docs.docker.com/build/dockerfile/frontend/

Given this `Dockerfile`:
```dockerfile
# syntax=docker/dockerfile:1.1.7
From alpine:3.13.5@sha256:1d30d1ba3cb90962067e9b29491fbd56997979d54376f23f01448b5c5cd8b462
```

Renovate should detect the dependency on `docker/dockerfile:1.1.7` and suggest an update to the current version (which is `docker/dockerfile:1.5.2` at the moment).

Renovate doesn't current do this, as evidenced by https://github.com/candrews/renovate-dockerfileVersions-test/issues/2

**Comment by rarkins** [comment]
Created feature request:

- #22804
